docs/README.md

```diff
@@ -5,12 +5,18 @@
 | [![Linux build status][1]][2]  | [![Windows build status][3]][4] |
 | [![Linux code coverage][5]][6] |                                 |
 
+| Interact with us!              |
+| :----------------------------- |
+| [![ort-talk][8]][7]            |
```

Comment:
One more final nit: Please make `7` go before `8` here and swap link definitions below accordingly.

Reply:
Adressed them.